Headers/AppLovinSDK.framework/Headers/MAReward.h

```diff
@@ -0,0 +1,64 @@
+//
+//  MAReward.h
+//  AppLovinSDK
+//
+//  Created by Thomas So on 8/10/18.
+//  Copyright © 2020 AppLovin Corporation. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * This object represents a reward given to the user.
+ */
+@interface MAReward : NSObject
+
+/**
+ * The label that is used when a label is not given by the third-party network.
+ */
+@property (nonatomic, copy, readonly, class) NSString *defaultLabel;
+
+/**
+ * The amount that is used when no amount is given by the third-party network.
+ */
+// [PLP] what are the units of "amount"?
```

Comment:
It's an arbitrary value defined by the publisher in our UI. However, that feature DNE anymore...

`defaultLabel` and `defaultAmount` are properties simply for pubs to compare against the `label` and `amount` properties - which, now, always returns an empty string `""` and `0`, respectively.